showcase/src/assets/translations/en.json

```diff
@@ -305,6 +305,14 @@
           "TITLE": "Based on Angular Material"
         },
         "TITLE": "Stark Typography"
+      },
+      "LAYOUT": {
+        "TITLE": "Stark Layout",
```

Comment:
Maybe this time we should just leave it as "Layout" because in this case this is not something Stark provides